docker/regtest/dockerfile-deps/joinmarket/latest/docker-entrypoint.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -e
+
+cd ..
+. jmvenv/bin/activate
+cd scripts
+
+export JM_onion_serving_host="$(/sbin/ip route|awk '/src/ { print $9 }')"
+
+
+# First we restore the default cfg as created by wallet-tool.py generate
+if ! [ -f "$CONFIG" ]; then
+    cp "$DEFAULT_CONFIG" "$CONFIG"
+fi
```

Comment:
Yes, if the file `/root/.joinmarket/joinmarket.cfg` does not exist, contents of `/root/default.cfg` will be copied.

Thinking about it, you are right, and it might be confusing that the `default.cfg` and `joinmarket.cfg` will be the same file because we copy it into the image before `python wallet-tool.py generate` is executed, which would create an "original default config file" for the server:
```
Created a new `joinmarket.cfg`. Please review and adopt the settings and restart joinmarket.
```
Keep in mind, this file will contain default values for interacting with mainnet.

Do you think it is better to copy the file afterwards, so `/root/default.cfg` is the "real" default config generated by joinmarket ?

Reply:
Yeah let's just keep everything as it is. Not really worth optimizing this further imo.